Lib/test/test_getpass.py

```diff
@@ -162,5 +167,54 @@ def test_falls_back_to_stdin(self):
             self.assertIn('Password:', stderr.getvalue())
 
 
+@unittest.skipIf(msvcrt is None, 'tests require system with msvcrt (Windows)')
+@unittest.skipUnless(support.MS_WINDOWS, "Windows-specific tests")
```

Comment:
I don't think we need second skip because `msvcrt` aviable only on windows, so we can use method like for `class UnixGetpassTest`:
```suggestion
@unittest.skipUnless(msvcrt, 'tests require system with msvcrt')
```